g2core/planner.h

```diff
@@ -174,8 +174,6 @@ typedef enum {                      // bf->buffer_state values in incresing orde
     MP_BUFFER_BACK_PLANNED,         // buffer ready for final planning; velocities have been set
     MP_BUFFER_FULLY_PLANNED,        // buffer fully planned. May still be replanned
     MP_BUFFER_RUNNING,              // current running buffer
-    MP_BUFFER_POLAND,               // Hitler used Poland as a buffer state
```

Comment:
Fair enough.  These two enums don't seem to be used anywhere anyway. :smile:

Guess Rob was in a weird mood or something when writing this. :confused: